### DISCUSSION TITLE
Input problem in unicorn

### DISCUSSION THREAD

**Discussion by soumah374** [django-commons/django-unicorn#619] [Q&A] [answered]
Hi I have a problem with unicorn when I type every time I type on the keyboard it sends a request to the server and this makes my application very slow. I need your help with a solution

**Comment by adamghill** [comment, marked as answer]
Have you looked at the `lazy` or `defer` modifiers: https://www.django-unicorn.com/docs/templates/#model-modifiers? That might be useful if you don't want to trigger an AJAX call on every key input. Closing this for now, but let me know you run into other problems.
